RedHatInsights/insights-operator-ldapauth
# Add Makefile & update README little bit

# Description

Add Makefile and update travis.yml file accordingly

Fixes # (issue)
CCXDEV-367

Please delete options that are not relevant.


## Testing steps
This should be tested automatically by Travis CI. Otherwise you can try to execute the make commands locally.
